src/lib/server/session-check.js

```diff
@@ -1,8 +1,9 @@
-import { getSession } from 'next-auth/react';
+import { getServerSession } from 'next-auth/next';
+import { authOptions } from '/src/pages/api/auth/[...nextauth]';
```

Comment:
I suggest we move the `authOptions` object in `/src/lib/server/auth.js` file and update the `[...nextauth].js` file accordingly.

Reply:
Sure, that makes sense.